pkg/api/validation/validation.go

```diff
@@ -1438,6 +1440,11 @@ func ValidatePersistentVolume(pv *api.PersistentVolume) field.ErrorList {
 		}
 	}
 
+	if pv.Spec.VolumeType != nil && len(*pv.Spec.VolumeType) > 0 {
```

Comment:
The validation should also check for the feature gate, since this is alpha.  See kube_features.go to add a new feature gate.